WordPress/gutenberg
# Allow dropping an image on an empty paragraph block to create an image block

## What?

Allow dropping files/blocks/HTML on an empty paragraph block.

## Why?


Fix https://github.com/WordPress/gutenberg/issues/29145.

## How?


I also created the `dragFiles` util to help test DnD in Playwright e2e tests.

## Testing Instructions




1. Create a new post or edit an existing post.
2. Add an empty paragraph block (note that the default block in a newly created post is not a paragraph block, yet).
3. Drag an image to the block.
4. It should show the placeholder upon hovering.
5. Dropping the image should create an image block.

## Screenshots or screencast 

https://user-images.githubusercontent.com/7753001/191423167-52a86f37-a6ea-4c3f-9b71-0fdfdbf5f72c.mp4
